### PR TITLE
made some updates to this page after feedback

### PR DESCRIPTION
**Pull request by idalithb**
Feedback would be appreciated: 
-should we remove Yarn v2+ command?
-step 8. "Query your subgraph" has some confusing information/language

### CODE REVIEW

File: website/pages/en/quick-start.mdx [hunk 1]
```diff
@@ -8,26 +8,26 @@ Ensure that your subgraph will be indexing data from a [supported network](/deve
...
 
 Go to the [Subgraph Studio](https://thegraph.com/studio/) and connect your wallet.
 
-Once connected, you can begin by clicking “Create a Subgraph”.
+Once your wallet is connected, you can begin by clicking “Create a Subgraph." It is recommended to name the subgraph in Title Case: "Subgraph Name Chain Name."
 
 ## 2. Install the Graph CLI
 
-The Graph CLI is written in JavaScript and you will need to have either `npm` or `yarn` installed to use it.
+The Graph CLI is written in TypeScript and you will need to have node.js and either `npm` or `yarn` installed to use it. Check that you have the most recent CLI version installed.
```
Comment (benface) [inline website/pages/en/quick-start.mdx:23]:
```suggestion
The Graph CLI is written in TypeScript and you will need to have `node` and either `npm` or `yarn` installed to use it. Check that you have the most recent CLI version installed. 
```

File: website/pages/en/quick-start.mdx [hunk 2]
```diff
@@ -36,7 +36,13 @@ Using [yarn](https://yarnpkg.com/):
 yarn global add @graphprotocol/graph-cli
 ```
 
-## 3. Initialize your Subgraph
+## 3. Initialize your subgraph from Existing Contract
```
Comment (benface) [inline website/pages/en/quick-start.mdx:39]:
```suggestion
## 3. Initialize your subgraph from existing contract 
```
Doesn't seem like title case is used.

File: website/pages/en/quick-start.mdx [hunk 3]
```diff
@@ -143,26 +145,26 @@ If your subgraph is failing, you can query the subgraph health by using the Grap
 }
 ```
 
-## 7. Publish Your Subgraph to The Graph’s Decentralized Network
+## 7. Publish your subgraph to The Graph’s Decentralized Network
 
-Once your subgraph has been deployed to Subgraph Studio, you have tested it out, and are ready to put it into production, you can then publish it to the decentralized network.
+Once your subgraph has been deployed to Subgraph Studio, you have tested it out, and it's ready to put it into production, you can then publish it to the decentralized network.
```
Comment (benface) [inline website/pages/en/quick-start.mdx:150]: Same comment as [here](https://github.com/graphprotocol/docs/pull/679#discussion_r1612014699).

Comment (idalithb) [reply website/pages/en/quick-start.mdx:150]:
```suggestion
Once your subgraph has been deployed to Subgraph Studio, you have tested it out, and you are ready to put it into production, you can then publish it to the decentralized network.
```

**Review by azf20 — APPROVED** [review]

**Pull request merged**